testnets/sgenetworktestnet/assetlist.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_id": "sgenetworktestnet",
```

Comment:
checks failed. This property should be "chain_name": "sgenetworktestnet", not the ID